lib/acts_as_taggable_on_mongoid/taggable.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+module ActsAsTaggableOnMongoid
+  # This module defines the class methods to be added to the Mongoid model so that
+  # tags can be defined on and added to a model.
+  #
+  # When a tag is added to a model, additional modules will be included to add methods that
+  # are needed only if a tag is actually being used.
+  module Taggable
```

Comment:
Copied from here:
https://github.com/mbleigh/acts-as-taggable-on/blob/master/lib/acts_as_taggable_on/taggable.rb

Changed it ti add parameters to all calls to define tag_lists individually rather than by class or globally.